### ISSUE TITLE
Datepicker Content is not within a landmark element

### ISSUE THREAD

**Issue by alon24** [carbon-design-system/carbon#7246]
Content is not within a landmark element
Violation
All content must reside within an element with a landmark role

![image](https://user-images.githubusercontent.com/5415443/98564414-07c97400-22b5-11eb-8097-f24fafa0c8c7.png)
![image](https://user-images.githubusercontent.com/5415443/98564480-1adc4400-22b5-11eb-8b3e-643f1333d27b.png)

**Comment by dakahn** [comment]
Good catch. This violation is due to the date picker dialog rendering outside of the landmark region on the page (React, amirite?). Adding `role="region"` to the dialog itself should fix this issue.

**Comment by alon24** [comment]
@dakahn any ETA on this ?
Or are you saying I have to do something in my code (This is run in a regular page not a dialog, so to which dialog do you mean?)

**Comment by alon24** [comment]
any updates on this issue?

**Comment by dhyliu** [comment]
@dakahn Are you going to fix the issue or are you expecting @alon24 to do some thing in his code and how? I'm focal responsible for making sure the accessibility support of applications running on IBM Cloud. We have some applications that apply the datepicker in their UI. I supposed this is likely not a unique issue impacting to single application. I'm looking forward to you quick advise. Thanks in advance.

**Comment by dakahn** [comment]
Circling back on this. Sorry for the lack of communication. The ticket is in our backlog and we do plan to get to it, but no one is assigned to it as of yet. Taking a quick look at this and determining if it's a fix with _Datepicker_ or if it's a fix with the underlying library we use to render our dialog. I'll post back when I have more information.

**Comment by dakahn** [comment]
After investigating the solution seemed simple enough. Pushing up a  PR now.
